2024-03-09 05:28:54 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trade-canopygrowth
2024-03-09 05:28:54 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 84, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 302, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x559a6dc90ec3 <unknown>
#1 0x559a6d988ce6 <unknown>
#2 0x559a6d9d3e48 <unknown>
#3 0x559a6d9d3f01 <unknown>
#4 0x559a6da173f4 <unknown>
#5 0x559a6d9f5edd <unknown>
#6 0x559a6da14899 <unknown>
#7 0x559a6d9f5c53 <unknown>
#8 0x559a6d9c6db3 <unknown>
#9 0x559a6d9c777e <unknown>
#10 0x559a6dc567fb <unknown>
#11 0x559a6dc5a815 <unknown>
#12 0x559a6dc44111 <unknown>
#13 0x559a6dc5b3a2 <unknown>
#14 0x559a6dc281ef <unknown>
#15 0x559a6dc7f4b8 <unknown>
#16 0x559a6dc7f6b3 <unknown>
#17 0x559a6dc90074 <unknown>
#18 0x7f6127294ac3 <unknown>